tests/integ/test_query_history.py

```diff
@@ -63,6 +65,9 @@ def test_query_history_no_actions(session):
     assert len(query_listener.queries) == 0
 
 
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC, reason="SNOW-533647: Support statement level parameters"
```

Comment:
If we don't do `alter session` then this might be supported. Maybe we should wait for `executemany` supporting statement level params.

Reply:
ok